backend/api/tests/contract-tests/package.json

```diff
@@ -8,8 +8,8 @@
     "buildjs": "cp -r assets/javascripts dist/assets/",
     "build": "npm run buildts && npm run buildjs",
     "publish": "node dist/provider/publish.js",
-    "verify": "node dist/provider/verifier.js",
-    "test": "npm run publish && jest --detectOpenHandles"
+    "test": "npm run publish && jest --detectOpenHandles",
+    "test:pact": "jest --detectOpenHandles"
```

Comment:
Think we can bin this `--detectOpenHandles` flag, [it's probably not useful here](https://jestjs.io/docs/cli#--detectopenhandles)